packages/unity-interface/portableExperiencesUtils.ts

```diff
@@ -50,7 +50,6 @@ function killPortableExperienceScene(sceneUrn: string) {
   if (peWorker) {
     forceStopParcelSceneWorker(peWorker)
     currentPortableExperiences.delete(sceneUrn)
-    store.dispatch(removeDebugPortableExperience(sceneUrn))
```

Comment:
@menduz this change is provoking the issue again:
1. I start a PEX from browser console:
```
spawnDebugPortableExperienceSceneFromUrn("urn:decentraland:off-chain:static-portable-experiences:portal-experience", "main")
```
2. The PEX is executed correctly.
3. I stop the PEX from Renderer doing the next calls to Kernel:
```
SetDisabledPortableExperiences <-- Passing a list with the PEX id. It will stop the PEX, and keep it as "denied".
SetDisabledPortableExperiences <-- Passing a list without the PEX id. It will extract the PEX from the list of "denied".
```
4. The PEX scene is stopped correctly.
5. I execute the step 1 again.
6. The PEX is no longer executed.